docs/user/service-catalog.md

```diff
@@ -0,0 +1,275 @@
+The services you are going to use are paid-for and, therefore, you need to have your
+account and billing settings set up properly.
+
+Once you have your account set up, follow the instructions at https://cloud.google.com/kubernetes-engine/docs/how-to/add-on/service-catalog/install-service-catalog. 
+You can skip the part of deploying the Service Catalog (which we already deployed
+using the Helm Chart). Although the instructions mention GKE, those should work
+in any Kubernetes cluster.
+
+To check that the broker has been succesfully deployed run the following:
+
+```
+kubectl get ClusterServiceBroker gcp-broker
+```
+
+If the Broker has been succesfully installed and the catalog has been properly
+downloaded you should get the following output:
+
+```
+NAME         URL                                                                                     STATUS   AGE
+gcp-broker   https://servicebroker.googleapis.com/v1beta1/projects/bitnamigcetest2/brokers/default   Ready    1m
+```
+
+## Kubeapps integration
+
+When a user clicks on Services Instances menu in Kubeapps they will get the list
+of Service Instances available in the selected namespace. As we haven't provisioned
+any yet, the list will be empty and we will get a message about provisioning
+an instance.
+
+![Empty Service Instances](../img/service-instances-empty.png)
+
+Clicking on "Deploy Service Instance" will provide us with a list of available
+classes:
+
+![Azure Classes](../img/azure-classes.png)
+![Google Classes](../img/google-classes.png)
+
+## Example 1. Wordpress with Azure MySQL as database
+
+For our first example we will provision an Azure MySQL instance and we will
+use it as the database for our new Wordpress deployment.
+
+### Provisioning the Azure MySQL database
+
+In the list of classes, click on "Select a plan" in the Azure Database for
+MySQL 5.7 card:
+
+![Azure MySQL class](../img/azure-mysql-class.png)
+
+We will select the `Basic Tier` which will be enough for our testing purposes
+
+![Azure MySQL basic-tier](../img/azure-basic-tier.png)
+
+After giving it a name, we will provide a set of options for our instance,
+like firewall rules to allow incoming traffic, the location of the instance,
+and the Azure resource group:
+
+![Azure MySQL options](../img/azure-mysql-options.png)
+
+Once we click on `Submit` the instance will start provisioning. Refresh the
+page regularly to check on the progress. Provisioning an Azure MySQL instance
+takes around 10-15 minutes.
+
+![Azure MySQL provisioning](../img/azure-mysql-provisioning.png)
+
+Once provisioned you will be able to create a Binding to connect your application
+to it. Click on `Add Binding` and select a name for it (the default should
+be OK). Creating a new binding should be very fast. 
+
+Once the binding creation has been completed, the details of the created
+secret can be explored directly from the same page, clicking on `show`.
```

Comment:
We should probably add a note here telling the user to note down the credentials, as we'll use them in the next step.